EIPS/eip-1271.md

```diff
@@ -127,12 +127,7 @@ Example implementation of a signing contract:
     }
 
     // Recover ECDSA signer
-    signer = ecrecover(
-      keccak256(abi.encodePacked("\x19Ethereum Signed Message:\n32", _hash)),
-      v,
-      r,
-      s
-    );
+    signer = ecrecover(_hash, v, r, s);
```

Comment:
Probably a dumb question, but is the old version broken in some way, or just less-than-ideal? From a quick read, it _looks_ like the code from before this PR would be secure, but would display a hash while signing instead of human readable text.

Reply:
I support this editorial change from @PhABC . 

I think the EIP-1271 is more general. 

Original text adding the "\x19Ethereum Signed Message:\n32" part of signature makes it confusing for reader. Some readers might deem such additional msg packing for `_hash` as necessary, but in fact it's not necessary in the spec of 1271.